build.sbt

```diff
@@ -31,8 +31,32 @@ inThisBuild(
 addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
 addCommandAlias("check", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
 
+// from this blog http://softwarebyjosh.com/2018/03/25/how-to-unit-test-your-dynamodb-queries.html
+lazy val copyJars = taskKey[Unit]("copyJars")
+
+def copyJarSetting(dir: String) =
+  Seq(
+    copyJars := {
+      import java.nio.file.Files
+      import java.io.File
+      // For Local Dynamo DB to work, we need to copy SQLLite native libs from
+      // our test dependencies into a directory that Java can find ("lib" in this case)
+      // Then in our Java/Scala program, we need to set System.setProperty("sqlite4java.library.path", "lib");
+      // before attempting to instantiate a DynamoDBEmbedded instance
+      val artifactTypes = Set("dylib", "so", "dll")
+      val files         = Classpaths.managedJars(sbt.Test, artifactTypes, update.value).files
+      Files.createDirectories(new File(s"$dir/native-libs").toPath)
+      files.foreach { f =>
+        val fileToCopy = new File(s"$dir/native-libs", f.name)
+        if (!fileToCopy.exists())
+          Files.copy(f.toPath, fileToCopy.toPath)
+      }
+    }
+  )
+
 val zioVersion       = "1.0.12"
 val zioConfigVersion = "1.0.6"
+val zioAwsVersion    = "4.0.4"
```

Comment:
`4.0.4` is not a real release but it was my attempt to publish snapshot releases from my ZIO2 branch. Now I see how it somehow leaked into both the README and the generated documentation.

The real latest release is `v3.17.87.0`. I will fix the site, sorry about it.

Reply:
Turns out that `4.0.4` is really the valid last release but my intention was to use `4.x` for the new ZIO2 release with many breaking changes :( I deleted these tags and next releases for now should continue to be `3.x`.

Reply:
No worries, I'll bump my version down. Thanks for bringing it up!